src/common/components/loadingOverlay/loadingOverlay.scss

```diff
@@ -11,11 +11,3 @@
   height: calc(100% + 20px);
   background-color: rgba(255, 255, 255, 0.7);
 }
-
-.loading-background > .loading-wrap {
-  position: static;
-}
-
-.error-block {
```

Comment:
I think .error-block can be removed. It was just for testing before we had actual styles form errors.